### PR TITLE
misc. browser test improvements

### PR DESCRIPTION
**Pull request by opqdonut**
Now that we have kaocha which captures stdout, it's fine to have more
verbose test logs: you only see them on failure. Verbose logs make
diagnosing CI failures easier.

### CODE REVIEW

**Comment by opqdonut** [comment]
Browser tests passed 5 times on circle. I'll rebase to remove the debug commit, and then this can go in IMO.

File: test/clj/rems/browser_test_util.clj [hunk 1]
```diff
@@ -171,6 +170,21 @@
...
+(def +character-delay+ 0.1)
+
+(defn fill-human [q text]
+  (doseq [c text]
+    (et/wait +character-delay+)
+    (et/fill (get-driver) q c))
+  (et/wait +character-delay+)
+  (let [value (get-element-attr q "value")]
+    (when-not (= text value)
+      (log/warn "Failed to fill field to" (pr-str text) "got" (pr-str value) "instead."))))
```
Comment (Macroz) [inline test/clj/rems/browser_test_util.clj:186]: I would like to see the typos and random delays here too. Is there a reason why you did not just copy them too?

**Review by Macroz — APPROVED** [review]
Looks good. Great catch!

File: test/clj/rems/browser_test_util.clj (hunk 1 above)
Comment (opqdonut) [reply test/clj/rems/browser_test_util.clj:186]: etaoin is under EPL, which is a copyleft license, so I didn't want to copy any code from it

Comment (opqdonut) [reply test/clj/rems/browser_test_util.clj:186]: I can reimplement typos if you want :)

Comment (Macroz) [reply test/clj/rems/browser_test_util.clj:186]: I think it would be nice.

**Review by Macroz — APPROVED** [review]

**Pull request merged**